### ISSUE TITLE
The paperThe paper mentions that Mobile MQA was used, but I couldn't find the implementation in your code.

### ISSUE THREAD

**Issue by KingTerminatora** [jiaowoguanren0615/MobileNetV4#3]

**Comment by jiaowoguanren0615** [comment]
Yes, I didn't add MQA block in this code, because I didn't find related code in [official code](https://github.com/tensorflow/models/blob/master/official/vision/modeling/backbones/mobilenet.py)

**Comment by KingTerminatora** [comment]
This is a non-official implementation code that I came across.[MobilenetV4](https://github.com/Reversev/Deeplearning_pytorch/blob/master/CV_net/MobileNetv4/MobileNetv4.py)

**Comment by jiaowoguanren0615** [comment]
Thank you so much! I will update my code and illustrate this statement
